psi4/src/psi4/lib3index/df_helper.h

```diff
@@ -56,38 +55,47 @@ class DF_Helper {
     DF_Helper(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> aux);
     ~DF_Helper();
 
-    // user options, must set before calling build()---------------------------------
-    // memory in doubles
+    // user options, must set before calling build() ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    // memory in doubles (defaults to 256,000,000) (2.048GB)
     void set_memory(size_t mem) { memory_ = mem;}
     size_t get_memory() { return memory_; }
 
-    // workflow method (store or direct)
+    // workflow method (store or direct) (defaults to STORE)
     void set_method(std::string met) { method_ = met;}
     std::string get_method() { return method_; }
 
-    // number of threads
+    // want the AO integrals in core? (defaults to FALSE) (I will adapt blocking to keep memory satisfied)
+    void set_AO_core(bool on) {AO_core_ = on;}
+    bool get_AO_core() { return AO_core_; }
+
+    // Do you want the MO integrals in core? (defaults to FALSE) (not my responsiblity to keep track of)
+    void set_MO_core(bool on) {MO_core_ = on;}
+    bool get_MO_core() { return MO_core_; }
+    
+    // number of threads (defaults to 1)
     void set_nthreads(size_t threads) { nthreads_ = threads;}
     size_t get_nthreads() { return nthreads_; }
 
-    // schwarz cutoff
+    // schwarz cutoff (defaults to 1e-12)
     void set_schwarz_cutoff(double cutoff) { cutoff_ = cutoff;}
     double get_schwarz_cutoff() { return cutoff_; }
-
-    // Do you want it outrageously fast?
-    void set_on_core(bool on) {core_ = on;}
-    bool get_on_core() { return core_; }
-
-    // metric power (defaults to -0.5)
+    
+    // metric power (defaults to -0.5) 
     void set_metric_pow(double pow) { mpower_ = pow;}
     double get_metric_pow() { return mpower_;}
 
+    // Want the metric to be held in core? (defaults to FALSE) (I will adapt blocking to keep memory satisfied)
     void hold_met(bool hold) {hold_met_ = hold;}
     bool get_hold_met() { return hold_met_;}
 
-    // tell me what the worst MO index size is!
+    // Tell me the worst MO size to improve blocking. (defaults to 0.5*AO_SIZE, must specify if greater!)
     void set_MO_hint(size_t wMO) {wMO_ = wMO;}
     size_t get_MO_hint() { return wMO_;}
-    // user options, must set before build---------------------------------
+    
+    // Enhanced memory use if (all Cleft = Cright). (defaults to FALSE!)
+    void set_JK_hint(bool hint) {JK_hint_ = hint;}
```

Comment:
Not sure what you mean.  If the procedure knows beforehand that all Cleft will be equal to all Cright for the J/K builds, then I will know at runtime to not declare two temporary buffers. Instead only one buffer is used and all buffers will get beefed up.